Backpack/Color/Classes/Generated/BPKColor.m

```diff
@@ -292,6 +292,17 @@ + (UIColor *)clear {
   return UIColor.clearColor;
 }
 
++ (UIColor *) label {
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
```

Comment:
This code is gonna be used a lot. Probably worth having a helper method that takes the two colours and spits back the right one.

Reply:
What this logic basically says is:
```
 - macro: if I'm compiling for iOS 13 then include the following code:
   - code if ios version is 13 then:
     - check whether dark mode is on or off 
   - end code if
 - end macro if
```

Reply:
Can we create a macro for this?

Reply:
You mean to create a macro for just for line 196, right?